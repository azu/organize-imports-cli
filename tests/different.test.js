import { test, expect } from '@jest/globals';
import runCli from "./run-cli.js";

test("--list-different - checks different files", async () => {
  const result = await runCli("cli/different", ["--list-different"]);
  expect(result.status).toMatchSnapshot();
  expect(result.stdout).toMatchSnapshot();
  expect(result.stderr).toMatchSnapshot();
  expect(result.write).toMatchSnapshot();
});
