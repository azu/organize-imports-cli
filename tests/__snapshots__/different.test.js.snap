// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`--list-different - checks different files 1`] = `0`;

exports[`--list-different - checks different files 2`] = `""`;

exports[`--list-different - checks different files 3`] = `""`;

exports[`--list-different - checks different files 4`] = `[]`;
